README.md

```diff
@@ -80,6 +80,8 @@ If you have any problems, submit an issue or send a message via gitter.
 
 ### CI
 
+We're using GitHub actions to run our CI process. View [a visualisation of the workflow here](https://lucid.app/lucidchart/invitations/accept/inv_74e30928-4106-49da-b75c-7a6291b225f4).
+
 #### Versioning
 
 Versions are auto incremented based on commit messages and SemVer (Major.Minor.Patch). When merging, please prepend your merge commit with:
```

Comment:
Hi! Please change to the following:

Version numbers will automatically increment based on commit messages and SemVer (Major.Minor.Patch). When merging, prepend your merge commit with the following: